### PR TITLE
Update for 1.0.4/1.1.1

### LINKED ISSUE #9: Re-enable precompilation on Julia 1.1
I'm not 100% sure but I believe all precompilation issues with Unitful are now fixed. See my comments at https://github.com/JuliaLang/julia/issues/29673 and also https://github.com/ajkeller34/Unitful.jl/pull/201

If it turns out https://github.com/JuliaLang/julia/pull/31047 impacts this, then the lower limit would be 1.2 (unless things are backported) but that issue does not seem to appear with the current versions of Unitful/PowerModelsAnnex/PowerSystemsUnits (with precompilation enabled).

### PR DESCRIPTION
**Pull request by iamed2**
Closes #9 

I think we should probably drop tests on Windows since there's no system code here. 

Still needs #4 before we can go 1.0.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+version = "0.2.0"
+
+[deps]
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
+
+[compat]
+Unitful = "0.14, 0.15"
+julia = "~1.0.4, 1.1.1"
```
Comment (ararslan) [inline Project.toml]: Is this package broken on 1.0.3 and earlier and on 1.1.0?

Comment (iamed2) [reply Project.toml]: The changes I made (around __init__ and precompilation) require https://github.com/JuliaLang/julia/pull/31047

The older version of the package should still work on older versions of Julia

File: src/utils.jl [hunk 2]
```diff
@@ -23,13 +23,6 @@ as they are treated separately.
 """
 asqtype(x::T) where {T<:Unitful.Units} = typeof(1.0*x)
 
-
-# Handle working with  Missings
-const UnitfulMissing = Union{<:Unitful.Quantity, Missings.Missing}
-
-*(y::Missings.Missing, x::T) where {T<:Unitful.FreeUnits} = y
```
Comment (nicoleepp) [inline src/utils.jl]: Why were these removed?

Comment (iamed2) [reply src/utils.jl]: The exist in Unitful now

Although I just checked and not until 0.15, so I need to change the Project.toml
